src/main/kotlin/no/nav/syfo/senoppfolging/v2/SenOppfolgingControllerV2.kt

```diff
@@ -84,6 +87,7 @@ class SenOppfolgingControllerV2(
             responseTime = response?.createdAt?.format(DateTimeFormatter.ofPattern("dd.MM.yyyy")),
             maxDate = sykepengerMaxDateResponse?.maxDate,
             gjenstaendeSykedager = sykepengerMaxDateResponse?.gjenstaendeSykedager,
+            shouldHaveAccessToSenOppfolging = hasValidUtsendtVarsel,
```

Comment:
Kanskje enda mer tydelig om man fjerner to "should have", siden "should" kan være litt vag. Kanskje bare "accessToSenOppfolging" eller "accessToForm"

Reply:
sounds good!